client/coral-admin/src/components/CommentList.js

```diff
@@ -81,12 +83,12 @@ export default class CommentList extends React.Component {
     const {commentIds} = this.props;
     const {active} = this.state;
     // check boundaries
-    if (active === null || !commentIds.size) {
-      this.setState({active: commentIds.get(0)});
-    } else if (direction === 'up' && active !== commentIds.first()) {
-      this.setState({active: commentIds.get(commentIds.indexOf(active) - 1)});
-    } else if (direction === 'down' && active !== commentIds.last()) {
-      this.setState({active: commentIds.get(commentIds.indexOf(active) + 1)});
+    if (active === null || !commentIds.length) {
+      this.setState({active: head(commentIds)});
+    } else if (direction === 'up' && active !== head(commentIds)) {
```

Comment:
I would recommend against using a library method with the native object already has a method that accomplishes the same thing. Does _.head() provide deeper functionality?